docs/source/tips-and-tricks.rst

```diff
@@ -25,50 +25,71 @@ Rxvt-unicode
 Terminus font and urxvt
 -----------------------
 
-The Terminus fonts does not have the powerline glyphs and unless someone submits 
-a patch to the font author, it is unlikely to happen.  However, Andre Klärner 
+The Terminus fonts does not have the powerline glyphs and unless someone submits
+a patch to the font author, it is unlikely to happen.  However, Andre Klärner
 came up with this work around: In your ``~/.Xdefault`` file add the following::
 
   urxvt*font: xft:Terminus:pixelsize=12,xft:Inconsolata\ for\ Powerline:pixelsize=12
 
-This will allow urxvt to fallback onto the Inconsolata fonts in case it does not 
+This will allow urxvt to fallback onto the Inconsolata fonts in case it does not
 find the right glyphs within the terminus font.
 
 Source Code Pro font and urxvt
 ------------------------------
 
-Much like the terminus font that was mentioned above, a similar fix can be 
+Much like the terminus font that was mentioned above, a similar fix can be
 applied to the Source Code Pro fonts.
 
 In the ``~/.Xdefaults`` add the following::
 
     URxvt*font: xft:Source\ Code\ Pro\ Medium:pixelsize=13:antialias=true:hinting=true,xft:Source\ Code\ Pro\ Medium:pixelsize=13:antialias=true:hinting=true
 
-I noticed that Source Code Pro has the glyphs there already, but the pixel size 
-of the fonts play a role in whether or not the > or the < separators showing up 
-or not. Using font size 12, glyphs on the right hand side of the powerline are 
-present, but the ones on the left don’t. Pixel size 14, brings the reverse 
+I noticed that Source Code Pro has the glyphs there already, but the pixel size
+of the fonts play a role in whether or not the > or the < separators showing up
+or not. Using font size 12, glyphs on the right hand side of the powerline are
+present, but the ones on the left don’t. Pixel size 14, brings the reverse
 problem. Font size 13 seems to work just fine.
 
+Urxvt unicode3 and frills
+-------------------------
+
+Make sure that, whatever urxvt package you're installing, both the `unicode3`
+and `frills` features are enabled at compile time. Run
+``urxvt --help 2>&1 | grep options:`` to get a list of enabled options.
+This should contain at least `frills`, `unicode3` and optionally `iso14755`
+if you want to input Unicode characters as well.
+
+Compiler flags example:
+
+.. code-block:: bash
+
+  --enable-frills \
+  --enable-unicode3
```

Comment:
And https://github.com/powerline/powerline/blob/029d54a64a6a3e4f2a8fd36948eea4cc0d37c5ba/docs/source/troubleshooting/linux.rst#L14. It may be extended with `--enable-frills` (WTF is this?) and command for determining configure options.